### DISCUSSION TITLE
Ability to customize / specify activity name in the monitoring hub

### DISCUSSION THREAD

**Discussion by asifkazi** [microsoft/Microsoft-Fabric-workload-development-sample#177] [Help] [answered]
In most cases it will be useful to customize / specify the activity name in the monitoring hub for jobs. Especially when there are multiple activities / tasks associated with an item. 

Is there a plan / eta to allow that level of customization.

For example, instead of just item name, it would be beneficial to say to have it say "Item Name - [File name Being Processed]" or "item name - task being performed" something of that sort so at a glance a user can tell what is going on. Right now, cannot really provide useful information in the monitoring hub for the activity

**Comment by yonadiv** [comment]
Thanks for your input, for now, this is not something that is in our backlog. I will pass it to our PM for his consideration.

**Comment by teddyberco** [comment, marked as answer]
Thanks for the suggestion, today there is customization option for each job to have "activity details" which opens a right pane which is populated by the workload iframe so any information could be added there.
Let me know if this make sense or customization on the job name itself is still required, maybe an additional column in the monitoring hub?
